components/brave_wallet_ui/common/async/__mocks__/lib.ts

```diff
@@ -74,3 +74,12 @@ export const findUnstoppableDomainAddress = async (address: string) => {
 export const getChecksumEthAddress = async () => {
   return {} as GetChecksumEthAddressReturnInfo
 }
+
+export const isStrongPassword = (value: string) => {
+  return (
+    (value.length >= 7) && // is at least 7 characters
+    /[-’/`~!#*$@_%+=.,^&(){}[\]|;:”<>?\\]/g.test(value) && // contains a special character
```

Comment:
Most likely `’”` should be replaced with `'"`

Reply:
updated